config/config.exs

```diff
@@ -0,0 +1,3 @@
+import Config
```

Comment:
Is this needed? We have configured default in code to 500 (RTCP Receiver l. 27). If we want to keep this I would use `Application.compile_env!` in RTCP Receiver.

Reply:
> If we want to keep this I would use Application.compile_env! in RTCP Receiver.

This is problematic as it would cause the compilation to fail in every packet that depends (even transitively) on it. In general, configs in libs should be avoided, thus originally I didn't have the config, just a module attribute. I've added it for testing purposes, as
1. I needed to access the value to not hardcode anything in tests
2. the test obviously time-sensitive and I expected issues with the CI, so I wanted to allow adjusting that
So you can consider it as a way of mocking the throttle for the test only

Do you think wrapping the config with an `if config_env == :test` would be better? Or adding a comment?

Reply:
PS I'm open for any other suggestions as I'm not super happy with that config either

Reply:
What about passing throttle_duration as an option to RTCP Receiver element?